include/TrustWalletCore/TWIoTeXStaking.h

```diff
@@ -15,23 +15,23 @@ TW_EXTERN_C_BEGIN
 TW_EXPORT_CLASS
 struct TWIoTeXStaking;
 
-/// Function to generate Stake message
+/// Function to generate Stake message.  Result must be freed.
 TW_EXPORT_STATIC_METHOD
```

Comment:
is this freeding must happen for iOS/Android interfaces or internal C++ code?

Reply:
let's remove this C interface instead? add a few staking message type to proto

Reply:
like Cosmos

Reply:
> is this freeding must happen for iOS/Android interfaces or internal C++ code?

In the swift&jni wrappers it is freed automatically, I think.

Reply:
if not necessary, let's remove this C interface? I can do that later after binary xcframework if you like